packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -951,6 +951,7 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>> + Clone> HoprCoreEthe
 
     async fn get_ticket_price(&self) -> Result<Option<U256>> {
         //utils_log::debug!("DB: get_ticket_price");
+        // TODO: use a default value if TICKET_PRICE_KEY value is not found
```

Comment:
It's not needed. If the ticket price key is not known, the function will return an `Option<U256>` with the value `None`. The code must handle that in the API.